[2024-08-07T14:20:20.568+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T14:20:20.606+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: airflow-space-exploration.Autorizacion_NASA scheduled__2024-08-05T00:00:00+00:00 [queued]>
[2024-08-07T14:20:20.621+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: airflow-space-exploration.Autorizacion_NASA scheduled__2024-08-05T00:00:00+00:00 [queued]>
[2024-08-07T14:20:20.622+0000] {taskinstance.py:2306} INFO - Starting attempt 3 of 3
[2024-08-07T14:20:20.640+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): Autorizacion_NASA> on 2024-08-05 00:00:00+00:00
[2024-08-07T14:20:20.650+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '***-space-exploration', 'Autorizacion_NASA', 'scheduled__2024-08-05T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/***-space-exploration.py', '--cfg-path', '/tmp/tmp0s53_5rh']
[2024-08-07T14:20:20.649+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=273) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-07T14:20:20.652+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask Autorizacion_NASA
[2024-08-07T14:20:20.652+0000] {standard_task_runner.py:64} INFO - Started process 282 to run task
[2024-08-07T14:20:20.722+0000] {task_command.py:426} INFO - Running <TaskInstance: airflow-space-exploration.Autorizacion_NASA scheduled__2024-08-05T00:00:00+00:00 [running]> on host 38873943cc86
[2024-08-07T14:20:20.835+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='***-space-exploration' AIRFLOW_CTX_TASK_ID='Autorizacion_NASA' AIRFLOW_CTX_EXECUTION_DATE='2024-08-05T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-05T00:00:00+00:00'
[2024-08-07T14:20:20.836+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T14:20:20.849+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T14:20:20.850+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "sleep 20 && echo 'OK' > /tmp/response_20240805.txt"]
[2024-08-07T14:20:20.861+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T14:20:40.864+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T14:20:40.868+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T14:20:40.898+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=***-space-exploration, task_id=Autorizacion_NASA, run_id=scheduled__2024-08-05T00:00:00+00:00, execution_date=20240805T000000, start_date=20240807T142020, end_date=20240807T142040
[2024-08-07T14:20:40.943+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-07T14:20:40.982+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T14:20:40.986+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
